### PR TITLE
Updated some research.csc.fi links

### PR DESCRIPTION
**Pull request by mradavi**
To fit with the newest structure of Pricing & Resources, research.csc.fi, I updated to new links:

- [Billing units (with billing models and calculator)](https://research.csc.fi/billing-units)

- [Quotas](https://research.csc.fi/quotas)

- [Purchasing](https://research.csc.fi/purchasing)

- [Pricing & Resources](https://research.csc.fi/pricing-and-resources)

### CODE REVIEW

**Comment by mlehtiva** [comment]
Looks good! 
One tiny comment is that in docs/computing/usage-policy.md, would we rather want to link to BUC?
[billing unit calculator](https://research.csc.fi/billing-units) -> https://research.csc.fi/billing-units#buc

**Review by mlehtiva — APPROVED** [review]
Approved :)

File: docs/cloud/pouta/accounting.md [hunk 1]
```diff
@@ -94,4 +95,4 @@ for them has been allocated.
 [Flavors]:vm-flavors-and-billing.md
 [Virtual machine lifecycle]:vm-lifecycle.md
 [TebiByte]:https://en.wikipedia.org/wiki/Tebibyte
-[Resource calculator]:https://research.csc.fi/billing-and-monitoring
+[Resource calculator]:https://research.csc.fi/billing-units
```
Comment (joonas-somero) [inline docs/cloud/pouta/accounting.md:98]:
```suggestion
[Billing unit and price calculator]:https://research.csc.fi/billing-units/#buc
```

File: docs/computing/usage-policy.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ GPU capacity compared to using CPUs or which can't be run on CPUs. In particular
 AI/ML workloads are prioritized, since many of them cannot be done at all on
 CPUs. A good rule of thumb is to compare the [billing unit (BU)](../accounts/billing.md)
 usage (_e.g._ with [`seff`](../performance/#quick-start-efficiency-report-with-seff)
-or the [billing unit calculator](https://research.csc.fi/pricing)) of the job on
+or the [billing unit calculator](https://research.csc.fi/billing-units)) of the job on
```
Comment (joonas-somero) [inline docs/computing/usage-policy.md:55]:
```suggestion
or the [billing unit calculator](https://research.csc.fi/billing-units/#buc)) of the job on
```

File: docs/data/datasets/datamanagement.md [hunk 3]
```diff
@@ -96,7 +96,7 @@ Budgeting and costing for your project is often dependent upon institutional res
     - Will you need additional paid help for organising and documenting data; are your data files, sheets etc. stored in uniform format and clearly named and well organised and understandable? Higher costs will apply if organising data has been neglected during the project. 
     - When you archive and open data, will there be costs involved?
 
-Read more about [CSC free-of-charge use cases and pricing](https://research.csc.fi/pricing).
+Read more about [CSC free-of-charge use cases and pricing](https://research.csc.fi/pricing-and-resources).
```
Comment (joonas-somero) [inline docs/data/datasets/datamanagement.md:99]: Looks like there is a more specific page for "free-of-charge use cases"

```suggestion
Read more about [CSC free-of-charge use cases and pricing](https://research.csc.fi/free-of-charge-use-cases).
```

**Comment by joonas-somero** [comment]
> Looks good! One tiny comment is that in docs/computing/usage-policy.md, would we rather want to link to BUC? [billing unit calculator](https://research.csc.fi/billing-units) -> https://research.csc.fi/billing-units#buc

I added this and a few others as suggestions.

File: docs/data/datasets/datamanagement.md (hunk 3 above)
Comment (joonas-somero) [reply docs/data/datasets/datamanagement.md:99]: > Looks like there is a more specific page for "free-of-charge use cases"

...but I now see that the link text has "and pricing" on it, so I'll leave it as it is.

**Pull request merged**